sechub-integrationtest/src/test/java/com/mercedesbenz/sechub/integrationtest/scenario10/PDSCodeScanSarifJobScenario10IntTest.java

```diff
@@ -98,11 +97,17 @@ public void a_user_can_start_a_pds_sarif_scan_and_get_the_sarif_results_transfor
 
         assertHTMLReport(htmlReport).
             containsAtLeastOneOpenDetailsBlock().
-            hasMetaDataLabel("quality-level", "high").
-            hasMetaDataLabel("test-label1", "Something special").
-            hasMetaDataLabel("test-label2", "").
-            hasMetaDataLabel("test-label3_with_html", "&lt;html&gt;HTML is allowed, but must always be escaped in reports!&lt;/html&gt;").
-            hasMetaDataLabel("test-label4_with_special_chars", "Line1\nLine2\tLine3");
+            hasHTMLString("<td><a href=\"#redCodeScanTable\">28</a></td>").
```

Comment:
You added your new tests - okay (of course)
But you dropped the existing meta data label tests. Why?

We need these tests to ensure that the labels are inside the report - it is an important version.

Please add the former meta data label checks/tests again.